backend/api/serializers/CreditTrade.py

```diff
@@ -146,9 +150,37 @@ def validate(self, data):
 
         return data
 
+    def save(self, **kwargs):
+        super().save(**kwargs)
+
+        if 'comment' in self.validated_data \
+                and self.validated_data['comment'] is not None\
+                and len(self.validated_data['comment'].strip()) > 0:
+            comment = CreditTradeComment(
+                credit_trade=self.instance,
+                comment=self.validated_data['comment'],
+                create_user=self.context['request'].user,
```

Comment:
Just for consistency, can you please do a request = self.context['request'] at the start of the function? Similar to how validate is.

Reply:
Sure